src/pages/messages/TasksPage.tsx

```diff
@@ -117,153 +111,159 @@ const TasksPage = () => {
   const completedCount = statusBuckets.find((bucket) => bucket.key === 'Completed')?.docCount;
   const closedCount = statusBuckets.find((bucket) => bucket.key === 'Closed')?.docCount;
 
+  const showSearchFilters = history.location.pathname === UrlPathTemplate.Tasks;
+
   return (
     <StyledPageWithSideMenu>
-      <SidePanel>
+      <SideMenu
+        expanded={showSearchFilters}
+        minimizedMenu={
+          <Link to={UrlPathTemplate.Tasks}>
+            <StyledMinimizedMenuButton title={t('common.tasks')}>
+              <AssignmentIcon />
+            </StyledMinimizedMenuButton>
+          </Link>
```

Comment:
Vurderte om SideMenu skulle kanskje tatt inn link, title og icon som prop i stedet, men tenker det kan være vi ønsker at det skal være annerledes en gang en plass, så lar det være for nå